awscli/argparser.py

```diff
@@ -16,6 +16,19 @@
 from difflib import get_close_matches
 
 
+HELP_BLURB = (
+    "To see help text, you can run:\n"
+    "\n"
+    "  aws help\n"
+    "  aws <command> help\n"
+    "  aws <command> <subcommand> help\n"
+)
+USAGE = (
+    "aws [options] <command> <subcommand> [parameters]\n"
```

Comment:
I'm not sure that's necessary -- couldn't subsequent subcommands be considered parameters of a parent command?